backend/test/integration/job_api_test.go

```diff
@@ -344,28 +344,34 @@ func (s *JobApiTestSuite) TestJobApis_noCatchupOption() {
 func (s *JobApiTestSuite) checkHelloWorldJob(t *testing.T, job *job_model.APIJob, experimentID string, experimentName string, pipelineVersionId string, pipelineVersionName string) {
 	// Check workflow manifest is not empty
 	assert.Contains(t, job.PipelineSpec.WorkflowManifest, "whalesay")
+
+	// Check resource references contain experiment and pipeline version.
+	resourceReferences := []*job_model.APIResourceReference{
+		{Key: &job_model.APIResourceKey{Type: job_model.APIResourceTypeEXPERIMENT, ID: experimentID},
+			Name: experimentName, Relationship: job_model.APIRelationshipOWNER,
+		},
+		{Key: &job_model.APIResourceKey{Type: job_model.APIResourceTypePIPELINEVERSION, ID: pipelineVersionId},
+			Name: pipelineVersionName, Relationship: job_model.APIRelationshipCREATOR},
+	}
+	assert.Len(t, job.ResourceReferences, 2)
+	assert.Subset(t, job.ResourceReferences, resourceReferences)
```

Comment:
Did a little searching, is this a better choice: https://godoc.org/github.com/stretchr/testify/assert#Assertions.ElementsMatch?

Reply:
done